lib/matplotlib/contour.py

```diff
@@ -1586,70 +1611,48 @@ def _initialize_x_y(self, z):
     _contour_doc = """
         Plot contours.
 
+        Call signature::
+
+            contour([X, Y,] Z, [levels], **kwargs)
+
         :func:`~matplotlib.pyplot.contour` and
         :func:`~matplotlib.pyplot.contourf` draw contour lines and
```

Comment:
Why does the `Axes.contour` documentation link to the `pyplot.contour` documentation (which is the same)?

Reply:
Given the discussion in #10974 it might be sensible to keep the links to pyplot as the pyplot function docs have a gallery below them.

Reply:
This same docstring is used both on the Axes and pyplot interfaces, because we don't want to maintain to almost identical versions. For this we have to pay the small price that both versions link to the same functions. We have the same issue with other passthrough functions. I've left the links as they are.